### DISCUSSION TITLE
Project SEO?

### DISCUSSION THREAD

**Discussion by smintz** [kvtools/valkeyrie#55] [Ideas]
Hey There! 
Great work on keeping the spirit of libkv alive!
In the past few months I found myself looking for this repo few times and it's just un-googleable... would love if we can make it a bit more discoverable... maybe put a deprecation notice in docker's libkv repo?

**Comment by ldez** [comment]
Hello,

I opened an issue on the libkv repository to "link" the 2 repositories.

https://github.com/docker/libkv/issues/222

I also added topics on the GitHub section.
